generator/test/good_test_data/primary_key.expected

```diff
@@ -30,6 +30,35 @@ class IntPK extends _IntPK with RealmEntity, RealmObject {
   }
 }
 
+class NullableIntPK extends _NullableIntPK with RealmEntity, RealmObject {
+  NullableIntPK(
+    int? id,
```

Comment:
Isn't it strange that nullable primary key is always expected as an argument even though it allows null? It is reasonable if only one null value is allowed in the collection. I suppose it is like that.

Reply:
Yeah - that's what I was thinking - I wanted people to explicitly specify `null` for the PK rather than think they can omit it.

Reply:
OK